ldap2pg/config.py

```diff
@@ -333,6 +333,7 @@ class Configuration(dict):
             'binddn': '',
             'user': None,
             'password': '',
+            'referrals': True,
```

Comment:
As @cbandy suggests, that would be cool to have 0 default, as suggested by python-ldap. Can you add a comment pointing to http://www.python-ldap.org/en/latest/faq.html and explaining the why on an alternative default value ?